Lesson 1/script.js

```diff
@@ -0,0 +1,166 @@
+// Homework
+// Task 1
+console.log('Task 1');
+let loop = (times=0,callback=null)=>{
+    if (typeof callback === "function"){
+        for (let i = 0; i<=times; i++){
```

Comment:
```suggestion
        for (let i = 0; i<times; i++){
```
в вашем случае цикл сработает 6 раз, а не 5